### PR TITLE
Cross-platform home directory detection

### PR DESCRIPTION
**Pull request by feross**
os.homedir() handles cross-platform complexities for us.

### CODE REVIEW

File: renderer/index.js [hunk 1]
```diff
@@ -20,6 +21,8 @@ var patch = require('virtual-dom/patch')
 
 var App = require('./views/app')
 
+var HOME = os.homedir()
```
Comment (ngoldman) [inline renderer/index.js]: nice, good to know!

**Comment by ngoldman** [comment]
:sparkles: lgtm :sparkles:

File: renderer/index.js [hunk 2]
```diff
@@ -318,7 +320,7 @@ function saveTorrentData (torrent) {
 function startTorrenting (torrentId) {
   var torrent = state.client.add(torrentId, {
     // use downloads folder
-    path: state.saved.downloads
+    path: state.saved.downloadPath
```
Comment (ngoldman) [inline renderer/index.js]: #73 might be because of a mismatch between `state.saved.downloadPath` and `torrent.path`

Comment (feross) [inline renderer/index.js]: I think I know what that issue is caused by. Will post a comment about it in a sec.

**Pull request merged**